pkg/common/prometheus/metrics.go

```diff
@@ -101,7 +119,8 @@ var (
 		// Possible voltype - "unknown", "block", "file"
 		// Possible optype - "create-volume", "delete-volume", "attach-volume", "detach-volume", "expand-volume"
 		// Possible status - "pass", "fail"
-		[]string{"voltype", "optype", "status"})
+		// Possible fault - "InvalidArgument", "NotFound", "ResourceInUse", "ManagedObjectNotFound", "CnsFault", "OtherFault"
```

Comment:
The line is a bit too long. Would you limit comments within 80 characters in each line?